jwql/website/apps/jwql/views.py

```diff
@@ -746,6 +747,50 @@ def jwqldb_table_viewer(request, tablename_param=None):
     return render(request, template, context)
 
 
+def log_view(request):
+    """Access JWQL monitoring logs from the web app.
+
+    Parameters
+    ----------
+    request : HttpRequest object
+        Incoming request from the webpage
+
+    Returns
+    -------
+    HttpResponse object
+        Outgoing response sent to the webpage
+    """
+
+    template = 'log_view.html'
+    log_path = get_config()['log_dir']
+    log_name = request.POST.get('log_submit', None)
+
+    full_uri = request.build_absolute_uri()
+
+    if 'dljwql' in full_uri:
+        server = 'dev'
+    elif 'tljwql' in full_uri:
+        server = 'test'
+    else:
+        server = 'ops'
+
+    full_log_paths = sorted(glob.glob(os.path.join(log_path, server, '*', '*')), reverse=True)
+    full_log_paths = [log for log in full_log_paths if not os.path.basename(log).startswith('.')]
+    log_dictionary = {os.path.basename(path): path for path in full_log_paths}
```

Comment:
Dictionary comprehension! I had forgotten that was a thing. :)

Reply:
This was super useful here.